Cheaper Effort/Middlewares/BrowserMiddleware.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+
+namespace Cheaper_Effort.Middlewares
+{
+    public class BrowserMiddleware
+    {
+        private readonly RequestDelegate _next;
+
+        public BrowserMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+
+        public Task Invoke(HttpContext httpContext, ILogger<DateLogMiddleware> logger)
+        {
+            var userAgent = httpContext.Request.Headers["User-Agent"].ToString();
+            var ipAddress = httpContext.Connection.RemoteIpAddress.ToString();
+            var url = httpContext.Request.Path;
+            logger.LogInformation("userAgent: " + userAgent);
+            logger.LogInformation("ipAddress: " + ipAddress);
+            logger.LogInformation("url: " + url);
+            //Debug.WriteLine("userAgent: " + userAgent);
```

Comment:
Don't leave unused code in comments in the future
![pepeOK](https://user-images.githubusercontent.com/68472824/206198824-9d9c010e-7c95-4848-a111-3f86a78082aa.png)